src/components/Pages/Guide/GuideMenu.js

```diff
@@ -10,7 +10,7 @@ function GuideMenuLink({ title, anchorTag, isHeading, isCurrentSection }) {
   // Each GuideSectionWrapper has an id={this.props.anchorTag}
   function goToSection(e) {
     history.pushState(null, null, `#${anchorTag}`);
-    document.getElementById(anchorTag).scrollIntoView(true);
+    document.getElementById(anchorTag).scrollIntoView({behavior: "smooth"});
```

Comment:
I removed the default  *smooth* behavior for scrolling in `coa.sass`. So, here i needed to add it to the guide page specifically. I removed it so that the document pagination didn't smooth scroll up the page when changing pages, which would take quite some time and be quite irritating when toggling through pages.